app/store/services/helpers/moment.js

```diff
@@ -1,8 +1,8 @@
 export function msToTime(duration = 0) {
-  let seconds = parseInt(duration / 1000 % 60, 10);
-  let minutes = parseInt(duration / (1000 * 60) % 60, 10);
-  let hours = parseInt(duration / (1000 * 60 * 60) % 24, 10);
-  let days = parseInt(duration / (1000 * 60 * 60 * 24) % 365, 10);
+  let seconds = parseInt((duration / 1000) % 60, 10);
```

Comment:
wtf????
parseInt((duration / 1000)
is that a cast
why?

Reply:
considering using moment lib!

Reply:
not my point

Reply:
got it! 
But my point was this would be removed asap!